gammapy/irf/energy_dispersion.py

```diff
@@ -67,15 +68,14 @@ def __init__(
     ):
         if interp_kwargs is None:
             interp_kwargs = self.default_interp_kwargs
-        axes = [
-            BinnedDataAxis(
-                e_true_lo, e_true_hi, interpolation_mode="log", name="e_true"
-            ),
-            BinnedDataAxis(
-                e_reco_lo, e_reco_hi, interpolation_mode="log", name="e_reco"
-            ),
-        ]
-        self.data = NDDataArray(axes=axes, data=data, interp_kwargs=interp_kwargs)
+
+        e_true_edges = np.append(e_true_lo, e_true_hi[-1]).value * e_true_lo.unit
+        e_true_axis = MapAxis.from_edges(e_true_edges, interp="log", name="e_true")
```

Comment:
Isn't it worth having a `MapAxis.lo` and a `MapAxis.hi` property?

Reply:
See comment above.